department-of-veterans-affairs/va.gov-cms
# VACMS-4989: Audience tags hardening.

## Description

See #4989.  

This shouldn't be merged until https://github.com/department-of-veterans-affairs/vets-website/pull/16983 is merged and we're ready to fix content on prod during the content release, particularly for the new CLP that's being launched on April 30. Probably after the [May 5 PW/CMS sync](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/product-team-support/product-team-weekly-syncs/public-websites-cms-weekly-sync.md). 


## Testing done

Manual. 


## Screenshots

<img width="1141" alt="Edit_Campaign_Landing_Page_Veteran_Trust_in_VA___VA_gov_CMS" src="https://user-images.githubusercontent.com/643678/116343820-3380dd80-a7b3-11eb-8189-f87974789fcc.png">


## QA steps

1. Go to https://pr5143-ii6l0ozrcxmdfu2c3fkwla2hsgqtvh0i.ci.cms.va.gov/node/18299/edit and add audiences from one of two controlled vocabularies. 


Validate that this is a non-breaking change for web. 
1. Go to https://web-ii6l0ozrcxmdfu2c3fkwla2hsgqtvh0i.ci.cms.va.gov/veteran-trust-in-va/  and see terms from controlled audiences (compare with http://preview-prod.vfs.va.gov/preview?nodeId=18299) ~**FAIL as of Wed April 28**~ _Pass as of April 29_. 
2. Note that when left empty web still generates the wrapper, 
![image](https://user-images.githubusercontent.com/643678/116344140-bb66e780-a7b3-11eb-980d-863856e59832.png)



### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
